java/src/jmri/jmrix/lenz/hornbyelite/messageformatters/EliteMessageFormattersBundle.properties

```diff
@@ -0,0 +1,7 @@
+# EliteMessageFormatterBundle.properties
+#
+# Message translation strings for the XpressNet Monitor from XNetMessages and XNetReplies
+
+# XNetReply Loco information
+XNetReplyLocoEliteSLabel                    = Elite Speed/Direction Information: Locomotive
+XNetReplyLocoEliteFLabel                    = Elite Function Information: Locomotive
```

Comment:
This file doesn't have an empty line at the end of the file.

Reply:
fixed